internal/util/reftracker/reftype/reftype.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2022 The Ceph-CSI Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package reftype
+
+import (
+	"encoding/binary"
+	"fmt"
+
+	"github.com/ceph/ceph-csi/internal/util/reftracker/errors"
+)
+
+// RefType describes type of the reftracker reference.
+type RefType int32
+
+const (
+	refTypeSize = 4
```

Comment:
do we need `int32` here ? or lesser size will also suffice ?

Reply:
Yeah I don't suppose there will be much more elements in the type, and `int8` should suffice, so I'll change it to that. Let me know if you disagree.

Reply:
sure, if we can suffice with lesser, its good to consume the same, at times little bit of buffer could also help, I am even fine between 8 and 32 ( if at all needed or we anticipate).  The 4 bytes looked heavy. so any lesser size I am fine .. :+1: